core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/transaction/Transaction.java

```diff
@@ -57,7 +60,7 @@
  * 
  * @author Jeen Broekstra
  */
-class Transaction {
+class Transaction implements AutoCloseable {
```

Comment:
Adding it helps with compiler and static analysis tools, and has no downsides as far as I know.